controllers/component_build_controller.go

```diff
@@ -278,20 +276,8 @@ func (r *ComponentBuildReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, nil
 	}
 
-	// when we are using already pipeline annotation we don't need anymore devfile
-	if pipelineRef == nil {
-		// Ensure devfile model is set
-		if component.Status.Devfile == "" {
-			// The Component has been just created.
-			// Component controller (from Application Service) must set devfile model, wait for it.
-			log.Info("Waiting for devfile model in component")
-			// Do not requeue as after model update a new update event will trigger a new reconcile
-			return ctrl.Result{}, nil
-		}
-	} else {
-		buildPipelineName, buildPipelineBundle, _ := getPipelineNameAndBundle(pipelineRef)
-		log.Info(fmt.Sprintf("Will use default pipeline from annotation %s : name: %s; bundle: %s", defaultBuildPipelineAnnotation, buildPipelineName, buildPipelineBundle))
-	}
+	buildPipelineName, buildPipelineBundle, _ := getPipelineNameAndBundle(pipelineRef)
```

Comment:
getPipelineNameAndBundle   checks ONLY if ref has resolver and that equals 'bundles'
since we are now generating pipelineRef we are adding resolver: bundles IN the code

also, it is called also in pac/simple code where it reads error

so it is pointless checking error here because resolver is hardcoded,
and also this way we can avoid additional  error handling code as is above this

Reply:
Well, it's safer to check it just for case if something would change in future

Reply:
removing it, it is used just for log message, and pac & simple are already logging that